packages/start-slicemachine/package.json

```diff
@@ -26,7 +26,7 @@
 	],
 	"scripts": {
 		"build": "vite build",
-		"dev": "vite build --watch",
+		"dev": "vite build --watch --mode development",
```

Comment:
No, it's only for this one because we rely on `import.meta` in the `start-slicemachine` script to detect development mode, adding this flag here allows us to tell Vite to keep that part of the code while developing instead of the production one :)